alternative_backend/boards/models.py

```diff
@@ -2,56 +2,58 @@
 
 
 class BoardCompany(models.Model):
-	description = models.CharField(max_length=500)
-	company_name = models.CharField(max_length=200)
+    description = models.CharField(max_length=500)
+    name = models.CharField(max_length=200)
+    code = models.IntegerField()
 
-	def __str__(self):
-		return '%s %s' % (self.id, self.company_name)
+    def __str__(self):
+        return '%s %s' % (self.id, self.name)
```

Comment:
Nie używaj formatowania poprzez symbol `%`. To jest stary format, który teraz utrzymywany jest tylko aby zapewnić kompatybilność wsteczną. Istnieje ryzyko, że może kiedyś zniknie.

używaj funkcji `.format()`